gap/OrbitalGraphs.gi

```diff
@@ -63,7 +63,7 @@ function(G)
         for iorb in innerorblist[i] do
             if not(orb[1] = iorb[1] and Size(iorb) = 1)
             then
-                graph := List([1..LargestMovedPoint(G)], x -> []);
+                graph := List(points, x -> []);
```

Comment:
This would work (and not give the error that I encountered) if this line was left as it was. I have done this now in PR #20.
```suggestion
                graph := List([1..LargestMovedPoint(G)], x -> []);
```